packages/components/src/tooltip/test/index.tsx

```diff
@@ -493,11 +463,6 @@ describe( 'Tooltip', () => {
 			expect(
 				screen.queryByRole( 'tooltip', { name: 'Inner tooltip' } )
 			).not.toBeInTheDocument();
-			expect(
-				screen.getByRole( 'button', {
-					description: 'Outer tooltip',
-				} )
-			).toBeVisible();
```

Comment:
Why was this assertion deleted?

Reply:
Same reason as above, see https://github.com/WordPress/gutenberg/pull/62947/files/4c022ed8d80fdce62795e3c0895fd3dfcbc08421..114badda5f68770118090baaf032c1a19b4cf84a#diff-eb9fcb45579ca92e96f172d835623c0682387b4ec0ff0e7ed5009b958b7c2c5c